jest.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  testEnvironment: 'jsdom',
```

Comment:
Without this, checks against the document (e.g., `if (!document)`) throw an exception